### PR TITLE
added github_markdown and github_wikipage shortcodes

### PR DESCRIPTION
**Pull request by olensmar**
these shortcodes make it easy to embed any GitHub .md or wiki page

### CODE REVIEW

**Comment by octalmage** [comment]
Thanks for the contribution! I'll add you to the readme and add branch support to the github_readme shortcode, good call!

File: github-readme.php [hunk 1]
```diff
@@ -13,38 +13,111 @@
...
+{
+    extract( shortcode_atts( array(
+        'user' => '',
+        'trim' => 0,
+        'cache' => 60,
+        'project' => '',
+        'page' => ''
+    ), $atts ) );
+
+    $transient="github_wikipate_" . $user . "_" . $project . "_" . $page;
```
Comment (octalmage) [inline github-readme.php:98]: Should this be `github_wikipage_`?

File: github-readme.php [hunk 2]
```diff
@@ -13,38 +13,111 @@
...
+        $html = Markdown::defaultTransform($markdown);
+        set_transient($transient, $html, $cache);
+    }
+    return $html;
+}
+
+function github_wikipage_func( $atts )
+{
+    extract( shortcode_atts( array(
+        'user' => '',
```
Comment (octalmage) [inline github-readme.php:91]: Hmm why did you use `user` and `project` instead of repo like the other shortcodes?

File: github-readme.php (hunk 1 above)
Comment (octalmage) [inline github-readme.php:98]: I'm going to go ahead and change it! 

Edit: Fixed in 56eba8cf5343576013bad16d39674c47dbe0101d.

Comment (olensmar) [inline github-readme.php:98]: perfect - thank you!

File: github-readme.php (hunk 2 above)
Comment (olensmar) [inline github-readme.php:91]: you're right - that would have been more consistent - please change this as well :-)

Comment (octalmage) [inline github-readme.php:91]: Cool just making sure I wasn't missing something!

Comment (octalmage) [inline github-readme.php:91]: Fixed in 14bbdc9bb9e5b01b8cf8d231b189e525b43944a2! Going to run a final test and push it live.

**Pull request merged**